cmd/githubrunnerscalesetlistener/main.go

```diff
@@ -71,23 +61,28 @@ func main() {
 		os.Exit(1)
 	}
 
+	if len(c.RunnerRepository) == 0 || len(c.RunnerOrg) == 0 || len(c.Token) == 0 || len(c.RunnerScaleSetName) == 0 {
+		fmt.Fprintln(os.Stderr, "GitHub config is not provided:", c.RunnerRepository, c.RunnerOrg, c.Token, c.RunnerScaleSetName)
+		os.Exit(1)
+	}
+
 	logger := logging.NewLogger(logging.LogLevelDebug)
 	ctrl.SetLogger(logger)
```

Comment:
Since we are not using runtime-controller, we don't need to import it just to set the logger right?